.changeset/neat-snails-kick.md

```diff
@@ -0,0 +1,5 @@
+---
+'@hyperdx/browser': patch
```

Comment:
we're breaking the import type, seems like we need to make this a minor? 

alternatively we can preserve the flag allowing it to no-op, but we should also make the changelog more descriptive that not only are we uninstalling the package, we're disabling the functionality completely.

Reply:
the flag should only be used internally and that's why I did't make it as a minor. keeping the arg sounds good and let me make the changelog more descriptive.